src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,7 +5,10 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int fuelConsuptionPerHour;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. The field `fuelConsuptionPerHour` should be private and accessed through getters and setters.